docs/getting-started/educator-pipeline/overview.md

```diff
@@ -0,0 +1,89 @@
+
+The Ed-Fi Alliance in November of 2021 released the EPDM Community and EPDM Core options for the EPP community. /wiki/spaces/GOV/pages/20334458 describes the additions and revisions identified to move a set of entities from the EPDM model into the Ed-Fi Standard- based on Ed-Fi EPP Starter Kit use cases. The  following provides a conceptual overview of the EPDM and the basic technology facts useful for those considering or starting a EPDM based implementation project:
+
+## Visual Overview of EPDM
+
+Below are visual representations of the EPDM model and its components. These images provide a conceptual understanding of the EPDM and its evolution over time.
+
+### Prior to November 2021
+
+![EPDM Prior to November 2021](https://edfidocs.blob.core.windows.net/$web/img/getting-started/epp/overview/tpdm_nov_2021.PNG)
+
+### Current State
+
+![EPDM Current State](https://edfidocs.blob.core.windows.net/$web/img/getting-started/epp/overview/epdm_current.png)
+
+## Which is best for my organization - EPDM-Core or EPDM-Community?
+
+| Choose EPDM-Core if... | Choose EPDM-Community if... |
+|------------------------|----------------------------|
+| I want to get started as quickly and simply as possible – using Ed-Fi EPP Starter Kits | I am currently using, or plan to use EPDM entities that are not currently in EPDM-Core |
+| My use cases focus on candidate demographics, candidate enrollment, or performance (candidate or others) | |
+
+If you are upgrading from TPDM v1.0 (and not using starter kits), or are unsure, start with TPDM-Community.
+
+## Installing EPDM
+
+| Installing EPDM-CORE | Installing EPDM-Community |
+|----------------------|---------------------------|
+| Install the Ed-Fi ODS/API platform v5.3 (defaults to including EPDM-Core extension) | EPDM-Community: Install the Ed-Fi ODS/API platform v5.3, but change the EPDM extension to be installed to EPDM-Community. This is done by following these [installation instructions](https://edfi.atlassian.net/wiki/spaces/EPP/pages/23169945/Installing+Ed-Fi+ODS+API+5.3+with+EPDM-Community+v1.1). |
+
+### Visual Representation
+
+#### EPDM-Core
+
+![EPDM-Core](https://edfidocs.blob.core.windows.net/$web/img/getting-started/epp/overview/epdm_core.png)
+
+#### EPDM-Community
+
+![EPDM-Community](https://edfidocs.blob.core.windows.net/$web/img/getting-started/epp/overview/epdm_community.png)
+
+More details about [TPDM-Core](https://edfi.atlassian.net/wiki/display/EFDS33/Educator+Preparation+Data+Model+Domain) and [TPDM-Community](https://edfi.atlassian.net/wiki/display/TPDMX/TPDM+Community?src=contextnavpagetreemode).
```

Comment:
For TPDM-Core, this should point to something more current. Again, this can be fixed in another iteration.

https://docs.ed-fi.org/reference/data-exchange/data-standard/model-reference/educator-preparation-data-model-domain/